dynamic_programming/Minimum Edit Distance.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+@details
+ Given two strings str1 & str2 and we have to calculate the minimum
+ number of operations (Insert, Delete, Replace) required to convert
+ str1 to str2.
+
+ @Author
+ Nirzak (github.com/nirzak)
+ */

```

Comment:
Please provide a brief description of what the algorithm is, and how it works, and a Wikipedia link (or another web source for algorithm explanation). You can see other algorithms such as [A star search](https://github.com/TheAlgorithms/C-Plus-Plus/pull/1468) to see how they are. 🙂
```suggestion
/**
 * @details
 * Given two strings str1 & str2 and we have to calculate the minimum
 * number of operations (Insert, Delete, Replace) required to convert
 * str1 to str2.
 *
 * @author [Nirzak](github.com/nirzak)
 */
```